account_vat_period_end_statement/views/report_vatperiodendstatement.xml

```diff
@@ -0,0 +1,251 @@
+                               <t t-set="total_to_pay" t-value="(total_to_pay + statement.interests_debit_vat_amount)"/>
+                           </tr>
+                           <!-- tax credit -->
+                           <tr>
+                               <td/>
+                               <td class="text-right">Tax credits</td>
+                               <td class="text-right">-<span t-esc="formatLang(env, statement.tax_credit_amount)"/></td>
+                               <!-- sum -->
+                               <t t-set="total_to_pay" t-value="(total_to_pay - statement.tax_credit_amount)"/>
+                           </tr>
+
+                           <!-- ...Other tot debit/credit -->
+                           <t t-foreach="statement.generic_vat_account_line_ids" t-as="generic_vat">
+                           <tr>
+                               <td class="text-right"><span t-esc="generic_vat.name"/></td>
+                               <td class="text-right"><span t-esc="generic_vat.account_id.name"/></td>
+                               <td class="text-right"><span t-esc="formatLang(env, generic_vat.amount*-1)"/></td>
+                               <!-- sum -->
+                               <t t-set="total_to_pay" t-value="(total_to_pay + (generic_vat.amount*-1))"/>
+                           </tr>
+                           </t>
+
+                           <!-- Total to pay -->
+                           <tr>
+                               <span t-if="(total_to_pay >= 0)">
+                                   <td/>
+                                   <td class="text-right"><strong>Total To Pay</strong></td>
+                                   <td class="text-right"><strong><span t-esc="formatLang(env, total_to_pay)"/></strong></td>
+                               </span>
+
+                               <span t-if="not (total_to_pay >= 0)">
+                                   <td/>
+                                   <td class="text-right"><strong>Total Credit</strong></td>
+                                   <td class="text-right"><strong><span t-esc="formatLang(env, (total_to_pay * -1))"/></strong></td>
+                               </span>
+                           </tr>
+                    </table>
+                    <!-- payment info -->
+                    <t t-if="statement.payment_ids">
+                        <br/>
+
+                    <table class="table table-condensed">
+                            <t t-foreach="statement.payment_ids" t-as="payment_line">
+                                <tr>
+                                    <!--td t-esc="Importo versato (Estremi del versamento: data ' + payment_line.date + payment_line.journal_id.name)" /-->
+                                    <td t-esc="'Importo versato (Estremi del versamento: data '+ time.strftime('%d/%m/%Y', time.strptime(payment_line.date, '%Y-%m-%d')) + ' - ' +payment_line.journal_id.name+')'" />
```

Comment:
Non ho capito se questa comparirà nel .pot, in caso andrebbe in inglese.
Inotlre c'è una riga commentata (da tenere?)

Reply:
@primes2h per pulizia, rimuovo la riga commentata.
Nel commit di migrazione ho anche rigenerato il POT e non c'è quindi non credo che ci finirà.